### PR TITLE
Deis release 1.7 post

### PR DESCRIPTION
**Pull request by mboersma**
Needs:
- [x] title
- [x] initial paragraph blurb
- [x] appropriate links in Features / Improvements / Under the Hood sections
- [ ] proofreading and feedback
- [x] change name of markdown file to match title

### CODE REVIEW

File: _posts/2015-06-05-deis-1-7.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+## What is Deis?
+
+Deis is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds upon [Docker](http://docker.io/) and [CoreOS](https://coreos.com/) to provide a lightweight PaaS with a Heroku-inspired workflow.
+
+## 1.7.0 Summary
+
+### New Features
+
+- Deis has adopted an [Open Roadmap]() and planning process
```
Comment (bacongobbler) [inline _posts/2015-06-05-deis-1-7.md]: this is linked below, but the link is http://docs.deis.io/en/latest/roadmap/roadmap/

File: _posts/2015-06-08-deis-1-7.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+- docker.service now depends on its mount to exist before starting
+- better documentation for Deis [backup/restore](http://docs.deis.io/en/latest/managing_deis/backing_up_data/) and for [CoreOS upgrades](http://docs.deis.io/en/latest/managing_deis/upgrading-deis/#upgrading-coreos)
+- `deisctl list` no longer lists an app named `deis-demo`, for example
+- [deis-swarm](http://docs.deis.io/en/latest/customizing_deis/choosing-a-scheduler/#swarm-scheduler) unit files are now fetched with `deisctl refresh-units`
+- [`deis config:push` and `config:pull`](http://docs.deis.io/en/latest/using_deis/config-application/#configure-the-application) validate the `.env` file syntax
+
+### Under the Hood
+
+- updated CoreOS to [647.2.0]()
+- `deis` honors requests environment variables such as `REQUESTS_CA_BUNDLE`
```
Comment (bacongobbler) [inline _posts/2015-06-08-deis-1-7.md]: request's

**Comment by bacongobbler** [comment]
reviewed it back and forth. LGTM

File: _posts/2015-06-08-deis-1-7-open-roadmap.md [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+
+Deis is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds upon [Docker](http://docker.io/) and [CoreOS](https://coreos.com/) to provide a lightweight PaaS with a Heroku-inspired workflow.
+
+## 1.7.0 Summary
+
+### New Features
+
+- Deis has adopted an [Open Roadmap](http://docs.deis.io/en/latest/roadmap/roadmap/) and planning process
+- Most components use [`alpine:3.1`](https://registry.hub.docker.com/_/alpine/) as a base, making Docker images smaller
+- Optional scripts allow for a [graceful cluster shutdown](http://docs.deis.io/en/latest/managing_deis/add_remove_host/#automatic-host-removal)
```
Comment (carmstrong) [inline _posts/2015-06-08-deis-1-7-open-roadmap.md]: I would call this "automatic host removal" as that's something that will bring people tears of joy and relief.

File: _posts/2015-06-08-deis-1-7-open-roadmap.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+title: "Deis v1.7 - Open Roadmap"
+author: carmstrong
+meta:
+  - name: description
+    content: Deis releases v1.7
+  - name: keywords
+    content: deis, release, v1.7, stable, production, production-ready, paas, private paas, heroku, github, docker, coreos, enterprise
+---
+
+The Deis project is excited to announce [v1.7.0](https://github.com/deis/deis/releases/tag/v1.7.0), featuring graceful EC2 cluster shutdown, smaller Docker images, and fixes for platform hardening. The leading open source PaaS is now yours to steer: read the Deis [Open Roadmap](http://docs.deis.io/en/latest/roadmap/roadmap/) and participate in [release planning](http://docs.deis.io/en/latest/roadmap/planning/#release-planning-meetings)!
```
Comment (carmstrong) [inline _posts/2015-06-08-deis-1-7-open-roadmap.md]: "Automatic host removal", and not just on EC2

Comment (mboersma) [inline _posts/2015-06-08-deis-1-7-open-roadmap.md]: > Automatic host removal

Ah yes, much better.

**Comment by carmstrong** [comment]
:ship:

**Pull request merged**